server/package.json

```diff
@@ -17,6 +17,7 @@
     "lib/*"
   ],
   "dependencies": {
+    "@babel/traverse": "^7.23.2",
```

Comment:
In general, we only want to include direct dependencies in our `package.json` file.

`@babel/traverse` is not a direct dependency. By looking in the `package-lock.json` file, I see that `@babel/traverse` appears to be a dependency of `@babel/core` which is a dependency of `istanbul-lib-instrument` which is a dependency of `nyc`. We directly use `nyc` for code coverage metrics in our `devDependencies`.

So, in fact, there are two things here. Since we only use `nyc` in our `devDependencies`, we don't want it included in the `dependencies` section. Second, if any one of `@babel/core`, `istanbul-lib-instrument` or `nyc` change their list of dependencies such that our use of `@babel/traverse` changes, we would want that to change automatically based on their dependency lists.

So I suggest that you remove this from your `package.json` file and instead run `npm audit fix` to update any necessary versions without adding new dependencies to your `package.json` file.

[I will say that sometimes there is no way around it and you will need to add a new dependency in `package.json`, but this should be a last resort.]

Reply:
Agree with Theresa here. Please remove the direct dependency of `@babel/traverse` in your PR; I believe it is updated in `package-lock.json`.